build.sbt

```diff
@@ -55,17 +55,20 @@ ReleaseKeys.publishArtifactsAction := PgpKeys.publishSigned.value
 
 libraryDependencies ++= Seq(
   "com.metamx" %% "scala-util" % "1.11.3" exclude("log4j", "log4j") force(),
-  "com.ircclouds.irc" % "irc-api" % "1.0-0014"
+  "com.ircclouds.irc" % "irc-api" % "1.0-0014",
+  "org.apache.kafka" % "kafka-clients" % "0.9.0.0",
+  "ch.qos.logback" % "logback-core" % "1.1.2",
```

Comment:
I figured you may have moved these to test before because they were interfering with the console output.. not sure how big of a problem it is to put them back here. I'm fine with either removing them or doing something like programmatically setting the logLevel to error only when outputting to the console to minimize log messages.